src/_P145_MQxxx.ino

```diff
@@ -73,7 +73,7 @@
 
 #define PLUGIN_145
 #define PLUGIN_ID_145     145           // plugin id
-#define PLUGIN_NAME_145   "Gases - MQxxx (MQ135 CO2, MQ3 Alcohol) [TESTING]" // "Plugin Name" is what will be dislpayed in the selection list
+#define PLUGIN_NAME_145   "Gases - MQxxx (MQ135 CO2, MQ3 Alcohol)" // "Plugin Name" is what will be dislpayed in the selection list
```

Comment:
Hmm comment seems to be an alcohol detector too :)

Reply:
It wasn't me, not sure if alcohol was involved though 🤣 
Corrected.